core_lib/src/interface/scribblearea.cpp

```diff
@@ -632,6 +633,8 @@ void ScribbleArea::mousePressEvent(QMouseEvent* e)
 
 void ScribbleArea::mouseMoveEvent(QMouseEvent* e)
 {
+    // Workaround for tablet issue (#677 part 2)
+    if (mStrokeManager->isTabletInUse() || !isMouseInUse()) { e->ignore(); return; }
```

Comment:
Actually we can't do this here, otherwise the canvas cursor won't be updated, leaving it stuck until you draw on the canvas again.